src/Result/tryCatch.spec.js

```diff
@@ -59,5 +59,13 @@ test('tryCatch functionality', t => {
   t.equals(good, 'Ok', 'returns an Ok when no error')
   t.equals(bad.message, msg, 'returns an Err with error on error')
 
+  const x = (a, b, c) => ({ a, b, c })
+
+  const noncurried = tryCatch(x, 1,2,3).either(identity, identity)
+  const curried = tryCatch(x, 1)(2)(3).either(identity, identity)
+
+  t.equals(equals(noncurried, { a: 1, b: 2, c: 3 }), true, 'preserves the arity of the passed function')
```

Comment:
instead of equals with an equals function checking for true, tape provides `same`.
But this is okay, I have done this a lot myself and it still is a vaild test